lib/intrinsic.js

```diff
@@ -48,9 +48,13 @@ intrinsic.cidr = (ipBlock, count, cidrBits) => {
  * key-value pairs.
  * @param {string} attr - The second-level key name, which is set to one of the
  * keys from the list assigned to key.
+ * @param {string?} defaultValue - The value that Fn::FindInMap will resolve to if the TopLevelKey and/or SecondLevelKey can not be found from the MapName map. This field is optional.
```

Comment:
Maybe want to add this to [the table in the top-level readme](https://github.com/mapbox/cloudfriend/blob/master/readme.md?plain=1#L19) for consistency but otherwise 👍

Reply:
Yes, good spot!